tests/extmod/uasyncio_heaplock.py

```diff
@@ -0,0 +1,31 @@
+try:
```

Comment:
`# test that asyncio.sleep_ms does not use the heap` ?

Reply:
ok